src/interfaces/ffi/core/mod.rs

```diff
@@ -12,6 +12,25 @@ pub use self::callbacks::*;
 /// fetched by batches of size [`NUMBER_OF_ENTRY_TABLE_LINE_IN_BATCH`].
 pub const NUMBER_OF_ENTRY_TABLE_LINE_IN_BATCH: usize = 100;
 
+#[repr(i32)]
+#[derive(Debug)]
+/// Callbacks return error codes, right now only 0 and 1 are specified.
+/// Other error codes will be forwarded to the client as a response of
+/// the main call error code so that the client can report some custom
+/// callbacks errors (for example the Flutter lib is using 42 to report
+/// an expection during a callback, save this exception and re-report this
+/// exception at the end of the main call if the response is 42).
+pub enum ErrorCode {
+    Success = 0,
+
+    /// :BreakingChange
```

Comment:
Is this really needed ?
What is the meaning of the leading `:` ?

Reply:
I used `:XXX` as a tag to search fast for something in a codebase (or link two part of the codebase together). The goal is to create an issue (I need to do it) for when we need to do a breaking change in Findex, add all these small stuff too… (currently there is only one occurrence but we could add more inside the codebase do not forget it during the next breaking release)

Reply:
Then I would prefer you create the issue before merging and remove this. Maybe you can ask @Manuthor what he thinks about it.

Reply:
Done in https://github.com/Cosmian/findex/issues/14